roles/install-k8s/templates/kubeadm.yaml.j2

```diff
@@ -52,6 +52,13 @@ containerRuntimeEndpoint: "unix:///run/containerd/containerd.sock"
 {% elif (runtime is defined) and 'crio' == runtime %}
 containerRuntimeEndpoint: "unix:///run/crio/crio.sock"
 {% endif %}
+{% if (swap_enabled is defined) and swap_enabled %}
+failSwapOn: false
+featureGates:
+  NodeSwap: true
```

Comment:
is this not beta yet? if yes - why in feature gate?

Reply:
Hi @mkumatag,
I'd followed this guide: https://kubernetes.io/blog/2023/08/24/swap-linux-beta/

Plus we do not require the featureGates to be set, it seems to work okay without it.
Thanks!